src/Yoti.Auth/ApplicationProfile.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+
+namespace Yoti.Auth
+{
+    /// <summary>
+    /// Profile of an application, with convenience methods to access well-known attributes.
+    /// </summary>
+    public class ApplicationProfile : Profile
+    {
+        public ApplicationProfile() : base()

```

Comment:
I don't think we want these constructors to be public.
Only we should be able to create instances of the Profiles + the Attributes.

Reply:
Yes good spot, will change